projects/1.16/assets/xaeros-minimap/xaerobetterpvp/lang/zh_cn.json

```
@@ -1,354 +1,552 @@
+	"gui.xaero_make_multi_automatic_msg2": "这将交换所选子世界和自动子世界的所有路径点数据，从而模拟使所选子世界自动化。请确保清楚你在做什么。",
+	"gui.xaero_make_multi_automatic_msg1": "你确定？",
+	"gui.xaero_make_automatic_msg2": "这将交换所选世界/服务器和自动世界/服务器的所有路径点数据，从而模拟使所选世界自动化。请确保清楚你在做什么。",
+	"gui.xaero_make_automatic_msg1": "你确定？",
+	"gui.xaero_options": "选项",
+	"gui.xaero_open_slime": "按键摁下时打开",
+	"gui.xaero_hide_world_names": "隐藏世界名称/IP",
+	"gui.xaero_waypoint_opacity_ingame": "游戏内路径点透明度",
+	"gui.xaero_subworld_dimension": "子世界/维度",
+	"gui.xaero_team": "团队",
+	"gui.xaero_block_transparency": "方块透明度",
+	"gui.xaero_terrain_depth": "地形深度",
+	"gui.xaero_entity_info_potion_effects": "药水效果（仅限单人游戏）",
+	"gui.xaero_entity_radar": "实体雷达",
+	"gui.xaero_patron_capes": "Patreon披风",
+	"gui.xaero_use_world_map": "使用世界地图区块",
+	"gui.xaero_world_map": "世界地图",
+	"gui.xaero_safe_mode_box": "使用基于Java（CPU）的等效模组而不是OpenGL（GPU）。以防正常模式失败，安全模式仅作为B计划。并非所有功能都在安全模式下工作。",
+	"gui.xaero_smooth_dots": "平滑点",
+	"gui.xaero_arrow_colour": "箭头颜色",
+	"gui.xaero_entity_info_armour": "护甲值",
+	"gui.xaero_entity_info_armour_numbers": "护甲数量",
+	"gui.xaero_teleport_default_command": "默认传送命令",
+	"gui.xaero_waypoints_distance": "最大路径点分布。",
+	"gui.xaero_box_entity_info": "显示你所指向的生物实体信息界面。",
+	"gui.xaero_delete_set_message2": "此设置将从当前世界中移除。",
+	"gui.xaero_minimap_guide": "点击拖放移动小地图。",
+	"gui.xaero_waypoint_names": "以上距离路径点名称",
+	"gui.xaero_show_entity_model": "显示实体模型",
+	"gui.xaero_show_flowers": "显示画",
+	"gui.xaero_show_full_amount": "物品全数",
+	"gui.xaero_entity_depth": "显示实体位置深度",
+	"gui.xaero_instant_waypoint": "快速路径点",
+	"gui.xaero_team_colours": "团队颜色",
+	"gui.xaero_false": "否",
+	"gui.xaero_true": "是",
+	"gui.xaero_yaw": "航向",
+	"gui.xaero_entity_info_numbers": "生命值",
+	"gui.xaero_reset_message2": "此操作无法复原。",
+	"gui.xaero_reset_message": "你确定你想重置默认设置吗？",
+	"gui.xaero_auto_map_size": "自动",
+	"gui.xaero_current_biome": "显示生物群系",
+	"gui.xaero_entity_info_max_hearts": "最大生命值",
+	"gui.xaero_entity_info_distance": "可达距离",
+	"gui.xaero_entity_info_stay": "显示实体信息",
+	"gui.xaero_entity_info_settings": "实体信息设置",
+	"gui.xaero_display_hostile": "显示敌对的生物",
+	"gui.xaero_compass_over_wp": "指南针上的路径点",
+	"gui.xaero_toggle_grid": "切换区块网格",
+	"gui.xaero_toggle_slime": "切换史莱姆区块",
+	"gui.xaero_entity_info": "实体信息",
+	"gui.xaero_toggle_waypoints": "切换路径点",
+	"gui.xaero_delete_set_message": "你确定你想删除当前设置吗",
+	"gui.xaero_delete_set": "删除设置",
+	"gui.xaero_waypoint_set": "设置",
+	"gui.xaero_world": "世界",
+	"gui.xaero_create_set": "新的路径点设置",
+	"gui.xaero_default": "默认",
+	"gui.xaero_clear_set_message2": "所有路径点将从设置中删除。",
+	"gui.xaero_clear_set_message": "你确定你想清除当前设置吗",
+	"gui.xaero_display_redstone": "显示红石",
+	"gui.xaero_all": "全部",
+	"gui.xaero_looking_at": "看到",
+	"gui.xaero_air_low": "低氧",
+	"gui.xaero_air_setting": "低氧提醒",
+	"gui.xaero_lighting": "灯光",
+	"gui.xaero_vanilla": "普通",
+	"gui.xaero_accurate": "准确",
+	"gui.xaero_block_colours": "方块颜色",
+	"gui.xaero_waypoints_scale": "标记大小",
+	"gui.xaero_show_distance": "距离标记",
+	"gui.xaero_used_seed": "种子使用:",
```

|Feedback Priority|
|---|
|🧗‍♀️ Boulder / **Blocking**|

```suggestion
	"gui.xaero_used_seed": "种子使用：",
```